integration-tests/container/perf_event_open/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu:jammy
+
+COPY sched_process_exit.c /sched_process_exit.c
+COPY entrypoint.sh /entrypoint.sh
+
+RUN apt update -y && apt install gcc -y
+RUN gcc sched_process_exit.c -o sched_process_exit
```

[nit] Run commands should be grouped together to reduce the number of layers in the images. Of course, this is not strictly required in this image.
```suggestion
RUN apt update -y && apt install gcc -y && \
    gcc sched_process_exit.c -o sched_process_exit
```

---

I would say, logically related commands have to be grouped together (you probably don't want to mix together a package installation and compilation phases). But as you said, for this image it's not important.